### PR TITLE
[15.0][MIG] web_widget_mpld3_chart

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
- Update Library Version
- Changed Js Scritps for Owl conventions
- Added BeautifulSoup dependency to make more standard implementation of widget on rendering
@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by JordiBForgeFlow** [comment]
@etobella can you merge this one?

File: web_widget_mpld3_chart/__manifest__.py [hunk 1]
```diff
@@ -6,11 +6,18 @@
...
     "summary": "This widget allows to display charts using MPLD3 library.",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
     "website": "https://github.com/OCA/web",
     "depends": ["web"],
-    "data": ["views/web_widget_mpld3_chart.xml"],
-    "external_dependencies": {"python": ["mpld3"]},
+    "data": [],
+    "external_dependencies": {"python": ["mpld3", "beautifulsoup4"]},
```
Comment (etobella) [inline web_widget_mpld3_chart/__manifest__.py]: You probably need to force the version of mpld3 to the one used on JS

**Comment by JordiBForgeFlow** [comment]
@etobella fixed

**Review by etobella — APPROVED** [review]
/ocabot migration web_widget_mpld3_chart
/ocabot merge nobump

**Pull request merged**